### ISSUE TITLE
Feature request slash command support

### ISSUE THREAD

**Issue by jokbon** [hyperdefined/TabCompleter#38]
Im trying to add tab complete for world edit
![image](https://github.com/hyperdefined/TabCompleter/assets/29400409/f28bbb4c-961d-43e2-991a-9d307c41a110)
![image](https://github.com/hyperdefined/TabCompleter/assets/29400409/1f767232-e07e-4798-9ffd-b85cbabbda00)

**Comment by hyperdefined** [comment]
I don't believe slash would count as a valid command. The plugin will only add commands that are registered to the server. Not sure if WorldEdit is doing something different for that command.
